templates/nginx/galaxyproject.j2

```diff
@@ -4,10 +4,59 @@ server {
 
 	server_name galaxyproject.eu;
 
+	# Redirect homepages to .org
+	location = / {
+		return 301 https://galaxyproject.org/eu/;
+	}
+	location = /freiburg/ {
+		return 301 https://galaxyproject.org/freiburg/;
+	}
+	location = /erasmusmc/ {
+		return 301 https://galaxyproject.org/erasmusmc/;
+	}
+	location = /belgium/ {
+		return 301 https://galaxyproject.org/belgium/;
+	}
+	location = /pasteur/ {
+		return 301 https://galaxyproject.org/pasteur/;
+	}
+	location = /genouest/ {
+		return 301 https://galaxyproject.org/genouest/;
+	}
+	location = /elixir-it/ {
+		return 301 https://galaxyproject.org/elixir-it/;
+	}
+	location = /ifb/ {
+		return 301 https://galaxyproject.org/ifb/;
+	}
+
+	# Redirect Galaxy server middle pane homepages to .org
+	location = /galaxy/ {
+		return 301 https://galaxyproject.org/bare/eu/usegalaxy/main/;
+	}
+	location = /index-metabolomics.html {
+		return 301 https://galaxyproject.org/bare/eu/usegalaxy/metabolomics/;
+	}
+	location = /index-proteomics.html {
+		return 301 https://galaxyproject.org/bare/eu/usegalaxy/proteomics/;
+	}
+
+	# Redirect other pages used by Galaxy servers to .org
+	location = /widgets/news.html {
+		return 301 https://galaxyproject.org/bare/eu/latest/news/;
+	}
+	location = /widgets/events.html {
+		return 301 https://galaxyproject.org/bare/eu/latest/events/;
+	}
+	location = /galaxy/news.html {
+		return 301 https://galaxyproject.org/bare/eu/news/;
+	}
+	location = /galaxy/events.html {
+		return 301 https://galaxyproject.org/bare/eu/events/;
+	}
+
+	# Everything else can keep being served by the European website, but under the github.io domain.
 	location / {
```

Comment:
But this you have above already, isn't it? Does this need to include here some wildcard?

Reply:
There are a lot of other pages not caught by the cases above. Those need to still be accessible, but we want to just change the domain. Or did I misunderstand your question?

Reply:
Don't 301 it, please just leave current proxying in place. I'd rather people don't end up at github.io, that's not a domain we want them to know exists.

Reply:
I think @bgruening wanted to move away from the galaxyproject.eu domain. The 301s above the catch-all send most traffic to galaxyproject.org, and people only end up at usegalaxy-eu.github.io for pages that aren't migrated yet.

Reply:
No, the plan is to redirect the entire galaxyproject.eu domain to org, but the URL should stay galaxyproject.eu. I think projects need that. However, pages that have not yet been migrated should be redirected to the GitHub page temporarily... Until we have migrated every feature and all.subpages. e.g the webhooks will still work but are redirected to the GitHub URL. Or old posts with the old links will still work until we have redirect for all imported posts to the new post. Does that make sense?

Reply:
Thanks for clarifying. And by GitHub URL you mean usegalaxy-eu.github.io? That was my understanding. This PR implements that.

Reply:
Yes. Does this sounds ok to you @hexylena ?

Reply:
> No, the plan is to redirect the entire galaxyproject.eu domain to org, but the URL should stay galaxyproject.eu. I think projects need that

so not redirecting but proxying?

> However, pages that have not yet been migrated should be redirected to the GitHub page temporarily...

again assuming you mean proxied here

If both of those are true, then, yes, agreed :)